src/components/Board.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const Board = () => {
```

Comment:
this should all be in data/board.js and to check the names you look at the board page